src/test/java/trackitnus/storage/JsonAdaptedContactTest.java

```diff
@@ -73,12 +73,12 @@ public void toModelType_invalidEmail_throwsIllegalValueException() {
         Assert.assertThrows(IllegalValueException.class, expectedMessage, contact::toModelType);
     }
 
-    @Test
-    public void toModelType_nullEmail_throwsIllegalValueException() {
-        JsonAdaptedContact contact = new JsonAdaptedContact(VALID_NAME, VALID_PHONE, null, VALID_TAGS);
-        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Email.class.getSimpleName());
-        Assert.assertThrows(IllegalValueException.class, expectedMessage, contact::toModelType);
-    }
+//    @Test TODO: modify this to be a positive test
+//    public void toModelType_nullEmail_throwsIllegalValueException() {
+//        JsonAdaptedContact contact = new JsonAdaptedContact(VALID_NAME, VALID_PHONE, null, VALID_TAGS);
+//        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Email.class.getSimpleName());
+//        Assert.assertThrows(IllegalValueException.class, expectedMessage, contact::toModelType);
+//    }
```

Comment:
I think we should create a github issue abt this so we dont forget about it. And include all the lines we commented out so we know what to replace.

Reply:
Actually in the upcoming PR there will be several more TODOs, it's a common & quick way to note that there are things todo haha since IntelliJ can highlight all todos in the project when you want to!